auth-web/src/util/config-helper.ts

```diff
@@ -127,6 +127,14 @@ export default class ConfigHelper {
     return `${process.env.VUE_APP_CORPORATE_ONLINE_URL}`
   }
 
+  static getSocietiesUrl () {
+    return `${process.env.VUE_APP_SOCIETIES_URL}`
+  }
+
+  static getCorpFormsUrl () {
+    return `${process.env.VUE_APP_CORP_FORMS_URL}`
```

Comment:
Was it decided that this URL is "close enough", or do you need a new key with the fragment to the forms?

Reply:
Confirmed with Janis. It's fine. No need to add a new one.